server/utils/nunjucksSetup.ts

```diff
@@ -70,5 +72,7 @@ export function registerNunjucks(app?: express.Express): Environment {
   njkEnv.addFilter('renderChooseBarcodeOptionRadiosFilter', renderChooseBarcodeOptionRadiosFilter)
   njkEnv.addFilter('renderChooseContactRadiosFilter', renderChooseContactRadiosFilter)
 
+  njkEnv.addGlobal('contactHelpdeskBannerExcludedOnPages', ['contact-helpdesk'])
```

Comment:
Set an array of page IDs that should not have the Contact Helpdesk banner displayed